frontend/src/api/index.ts

```diff
@@ -171,10 +171,6 @@ export async function getValues(
     `${api}/stats/${level}/${category}/fips-value?` + params,
   );
 
-  /** suppress age adjusted data for privacy */
-  if (["cancerincidence", "cancermortality"].includes(category))
-    data.values = omitBy(data.values, (value) => value.value < 3);
-
```

Comment:
To support removing this, here's the official word on suppression from our only source of cancer incidence/mortality data, State Cancer Profiles: https://statecancerprofiles.cancer.gov/suppressed.html.

I was going to mention to you that `value` for cancer models is the annual average rate, not the count, so I could imagine this filter being a problem, but there's no problem now that you've removed it! 🙂